File: playground/boj_10250/boj_10250.py
'''
----------------------------------------------
Sub : [BOJ] ACM호텔
Link: https://www.acmicpc.net/problem/10250
Tag : Python
Memo
----------------------------------------------
'''
 
T = int(input())  # 테스트 케이스 개수 입력

def sol(H, W, N):
    number = 1;
    for i in range(W):
        for j in range(H):
            if number == N:
                print((j+1)*100 + (i+1))
            number += 1


for _ in range(T):
    H, W, N = map(int, input().split())  # 호텔의 층 수, 각 층의 방 수, N번째 손님
    sol (H, W, N)

File: playground/BOJ/boj_10250/boj_10250.js
/*
----------------------------------------------
Sub : [BOJ] ACM호텔
Link: https://www.acmicpc.net/problem/10250
Tag : JS, 
Memo
----------------------------------------------
*/

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const T = Number(input[0]);

const sol = (H, W, N) => {
  let number = 1;
  for (let i = 0; i < W; i++) {
    for (let j = 0; j < H; j++) {
      if (number === N) {
        console.log((j + 1) * 100 + (i + 1));
        return;
      }
      number += 1;
    }
  }
};

for (let i = 1; i <= T; i++) {
  let [H, W, N] = input[i].split(" ").map(Number);
  sol(H, W, N);
}
